x-pack/plugins/security_solution/server/lib/siem_migrations/rules/task/agent/sub_graphs/translate_rule/nodes/ecs_mapping/ecs_mapping.ts

```diff
@@ -59,7 +59,7 @@ export const getEcsMappingNode = ({
 };
 
 const getTranslationResult = (esqlQuery: string): SiemMigrationRuleTranslationResult => {
-  if (esqlQuery.match(/\[(macro|lookup):[\s\S]*\]/)) {
+  if (esqlQuery.match(/\[(macro):[\s\S]*\]/)) {
```

Comment:
No need to address this now, but:
Should we also take into account the `[logs_index_pattern]` placeholder that we decided to use when no integration matched? Or, wouldn't it be better to just do a syntax check, to assign a translation result?

Reply:
We could yeah, but the index pattern one has to happen as the last action after all validation because we need to test the query syntax. The detection and return of partial result is done in its own node added in the PR merged yesterday currently, so it is working at least. We can change it later if you want :1+